### ISSUE TITLE
Exiting Xemu at random points will hang Xemu or slow down the close time returns

### ISSUE THREAD

**Issue by AkihitoZero5454** [xemu-project/xemu#1583]
### Bug Description

The hang which thought was fixed came back again
https://github.com/xemu-project/xemu/issues/1391

### Expected Behavior

Xemu closing immediately

### xemu Version

v0.7.118

### System Information

Windows 10
AMD Ryzen 7 3700X 8 Core Processor
Nividia Geforce GTX 1070 Ti

**Comment by efrainm23** [comment]
Yes, my last post on the previous issue indicated that it had improved at that time. It does still seem to be happening just not the same degree as before. As I noted in my post on the other issue, it does seem to take longer to close the first time you open and close the emulator. However, subsequent opening and closing of the emu seems to work normally. 

In my previous post, I had been running the emulator on a traditional spinning disk HDD, which may have contributed to the slowness when closing the emulator. It is now on an SSD and it closes much faster when the issue does occur. It might hang now for 10 or less while before it might have hung for about a minute. 

So it does seem improved but the issue is definitely still there to some degree.

**Comment by abaire** [comment]
Please close this as a dup of #1391 - there is no need to create a new issue for an issue that is already open, you can just add a comment there indicating it was not resolved.

Also worth looking into whether this is a dup of #1431 which seems to indicate that an update check might be at fault.

**Comment by Triticum0** [comment]
Closed
